seatunnel-e2e/seatunnel-connector-v2-e2e/connector-prometheus-e2e/src/test/resources/prometheus_remote_write.conf

```diff
@@ -0,0 +1,66 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+######
+###### This config file is a demonstration of streaming processing in seatunnel config
+######
+
+env {
+  parallelism = 1
+  job.mode = "BATCH"
+}
+
+source {
+  FakeSource {
+    schema = {
+      fields {
+        c_map = "map<string, string>"
+        c_double = double
+        c_timestamp = timestamp
+      }
+    }
+    result_table_name = "fake"
+    rows = [
+       {
+         kind = INSERT
+         fields = [{"__name__": "test1"},  1.23]
+       },
+       {
+         kind = INSERT
+         fields = [{"__name__": "test2"},  1.23]
+       }
```

Comment:
I did a test here. You can try to use long type to test(`c_timestamp = long` eg：1726077004873 ). This way can control the time to test for range query

Reply:
cc: @hailin0

Reply:
idea is wrong, ignore